### ISSUE TITLE
The variables are not in Chinese

### ISSUE THREAD

**Issue by delgh1** [twitter/the-algorithm#916]
**Describe the bug**
The variable names are not in Chinese. You can make all variable names' length be only two bytes long by using Chinese characters.

**To Reproduce**
1. Go to 'The hub'
2. Click on 'Twitter'
3. Scroll down to 'The thing'
4. See error: no Chinese

**Expected behavior**
All variable names are in Chinese.

**Screenshots**
Example:
![68747470733a2f2f692e6c6f6c692e6e65742f323031392f30372f30382f3564323234623637346236336234333536342e706e67](https://user-images.githubusercontent.com/60296981/229299301-1a273e1d-19d0-40a3-9497-7948e33997f0.png)

**Environment**
The earth

**Additional context**
香風卜麼弟正神象得條抄流肉這卜止八向米竹，像只流松、息而科的朵跟手意經黃幾，住貝雄黑面愛只娘什只尼唱回弓瓜魚林員至；村央節夕婆院枝月：話七白借你刃七八青麻吹出蝴河黃，刃的星穿。

害汁動游，條女頁布蝸像，水吧詞人春千泉兒室哥車魚汁，點起每兌古雨未幼。占她有加次收斤才，叫鳥飽！定見科飯方送辛物王花綠法刃結看古牛。

**Comment by nobodyindustries** [comment]
LGTM, let's save those bytes

**Comment by RAKtheUndead** [comment]
Discriminatory towards 8-bit computers; each ideograph takes at least two bytes to encode and therefore takes additional time to process on older computers, unlike ASCII.

https://en.wikipedia.org/wiki/CJK_Unified_Ideographs

**Comment by delgh1** [comment]
> Discriminatory towards 8-bit computers; each ideograph takes at least two bytes to encode and therefore takes additional time to process on older computers, unlike ASCII.
> 
> https://en.wikipedia.org/wiki/CJK_Unified_Ideographs

True, but instead of unicode, we can use chinese telegraphic code, reject modernity and embrace tradition

[](https://en.wikipedia.org/wiki/Chinese_telegraph_code)

![440px-Obsolete_chinese_telegraph_code](https://user-images.githubusercontent.com/60296981/229299771-943404fb-1265-4dce-b9f6-f59ed465d874.jpg)

**Comment by RAKtheUndead** [comment]
Requires additional resolution, therefore also discriminating against 8-bit computers which do not typically have the screen resolutions required to render such characters legibly.

Maybe we can come to a middle ground and choose Hangul.

**Comment by delgh1** [comment]
> Requires additional resolution, therefore also discriminating against 8-bit computers which do not typically have the screen resolutions required to render such characters legibly.
> 
> Maybe we can come to a middle ground and choose Hangul. Other options are Ethiopic and Linear B.

An alternative would be using [ISO-2022-JP](https://en.wikipedia.org/wiki/ISO-2022-JP), aka JIS, for Japanese. 7 bit, should be a drop in replacement for ASCII

**Comment by dnlsndr** [comment]
This has gone too far. The amounts of bytes that are wasted in this repo... kids in africa could eat that. shm wwjd
